src/remote.rs

```diff
@@ -0,0 +1,144 @@
+use smash_arc::Hash40;
+
+pub mod arc;
+pub mod table;
+
+static USAGE: &'static str = 
```

Comment:
Not a requested change but leaving a note here that this should eventually be replaced by structopt.